### ISSUE TITLE
a11y: keyboard navigation is not hovering alerts in user retention settings

### ISSUE THREAD

**Issue by IsaSih** [rancher/dashboard#13425]
**Setup**
- Rancher version: 
Rancher | v2.11-a1bda936c809e0394520c9d40d2998529223401e-head
Dashboard | master 8795891
Rancher CLI | v2.11.0-rc.1

**Describe the bug**
In the user retention settings page, the keyboard navigation is not selecting the warning icons of the disable/delete `inactivity period` fields and `User Retention Process Schedule`

**To Reproduce**
1- From the burguer menu, select Users & Authentication
2- Use the keyboard navigation to navigate the User retention settings feature 
3- Try to hover and read the instruction warnings of the disable/delete `inactivity period` fields and `User Retention Process Schedule`

**Result**
the keyboard navigation is not selecting the warning icons of the disable/delete `inactivity period` fields and `User Retention Process Schedule`

**Expected Result**
The keyboard navigation should be able to select the disable/delete `inactivity period` fields and `User Retention Process Schedule` to display the instructions in the warning messages that are displayed by hovering those icons

**Screenshots**

https://github.com/user-attachments/assets/5f4fcf2c-c620-4af4-b257-6b9483cd3e80

**Comment by aalves08** [comment]
Closing as it's a duplicate of #12807 , which will be fixed by https://github.com/rancher/dashboard/pull/13416

FYI @rak-phillip

**Comment by aalves08** [comment]
https://github.com/user-attachments/assets/b9aa81a2-aa2e-44af-8383-133214d04648
